### PR TITLE
Abdul qh 2 identity evidence tsv validation

### PR DESCRIPTION
**Pull request by abdul-sathar**
The test case is: 

- Create test data for event
- Push to Kinesis Stream
- Validates TSV row and column in Identity evidence through Athena

### CODE REVIEW

File: constants/identity-evidence-data.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**
```
Comment (jankoulaga) [inline constants/identity-evidence-data.js:1]: We usually don't add this... It would be better to provide some js-doc instead.

**Review by jankoulaga — COMMENTED** [review]
@abdul-sathar this looks messy... It's hard to review when a lot of the diff is covered by https://github.com/LiveIntent/castle/pull/1/files. Let's try to get that one merged, because it's hard to focus on any of the details of this PR.

File: constants/identity-evidence-data.js (hunk 1 above)
Comment (abdul-sathar) [reply constants/identity-evidence-data.js:1]: Ahh ok, I will remove it. Thank you for your input.

**Comment by jankoulaga** [comment]
@abdul-sathar you need to resolve conflicts 👍

**Comment by jankoulaga** [comment]
@abdul-sathar from what i see, this has not been updated since this morning. Do you need help merging?

**Comment by jankoulaga** [comment]
@abdul-sathar can you please resolve merge conflicts? How can this be reviewed if half of the code is un-needed?

**Comment by abdul-sathar** [comment]
> @abdul-sathar can you please resolve merge conflicts? How can this be reviewed if half of the code is un-needed?

Hi Janko, thank you for your message. I have resolved the conflicts and also merged with Master and also I have removed the review label, as its still in progress now, facing some issues which need to address.

**Pull request closed without merging**